src/utils.js

```diff
@@ -180,6 +181,64 @@ export function buildLocaleDateAndTime(dateTime, timezone, locale) {
     .format('L LT');
 }
 
+export function getSelectedSlipData(pickSlipsData, selectedRequestId) {
+  const sel = pickSlipsData.filter((pickSlip) => {
+    return pickSlip['request.requestID'] === selectedRequestId;
+  })[0];
+  if (sel === undefined) {
+    return [];
+  } else {
+    return [sel].flat();
+  }
+}
+
+export function getSelectedSlipDataMulti(pickSlipsData, selectedRows) {
+  const sel = pickSlipsData.filter((pickSlip) => {
```

Comment:
If filter does not find any element that fits your condition it returns empty array [] instead of undefined.
As a result if (sel === undefined) never happen.